README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop)](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io)
```

Comment:
![image](https://user-images.githubusercontent.com/8705386/28499628-9b957ff6-6fd7-11e7-8024-1a579043338e.png)
Right Now `build passing` thing coming at starting of the readme file.
Please move this below ` Open Source Help Community`
so in order to do that swap line 1 and line 2